PAWS/PAWSStandard.swift

```diff
@@ -85,17 +85,17 @@ actor PAWSStandard: Standard, EnvironmentAccessible, HealthKitConstraint, Onboar
         do {
             try await upload(sample: electrocardiogram)
             
-            supplementalMetrics.append(contentsOf: try await electrocardiogram.precedingPulseRates)
-            supplementalMetrics.append(contentsOf: try await electrocardiogram.precedingPhysicalEffort)
-            supplementalMetrics.append(contentsOf: try await electrocardiogram.precedingStepCount)
-            supplementalMetrics.append(contentsOf: try await electrocardiogram.precedingActiveEnergy)
+            supplementalMetrics.append(contentsOf: (try? await electrocardiogram.precedingPulseRates) ?? [])
+            supplementalMetrics.append(contentsOf: (try? await electrocardiogram.precedingPhysicalEffort) ?? [])
+            supplementalMetrics.append(contentsOf: (try? await electrocardiogram.precedingStepCount) ?? [])
+            supplementalMetrics.append(contentsOf: (try? await electrocardiogram.precedingActiveEnergy) ?? [])
             
-            if let precedingVo2Max = try await electrocardiogram.precedingVo2Max {
+            if let precedingVo2Max = try? await electrocardiogram.precedingVo2Max {
                 supplementalMetrics.append(precedingVo2Max)
             }
             
             for supplementalMetric in supplementalMetrics {
-                try await upload(sample: supplementalMetric)
+                try? await upload(sample: supplementalMetric)
             }
```

Comment:
@PSchmiedmayer, should each iteration of the loop have its own `do`/`catch` with a log statement, or is this fine?

Reply:
Sure; that would also be a good approach to add some logs in case elements are failing. If we use `try?` it will never run into the error state.